vm/chef/cookbooks/django/recipes/default.rb

```diff
@@ -24,3 +28,33 @@
     pip3 install django
 EOH
 end
+
+execute 'enable_apache_modules' do
+  command 'a2enmod proxy proxy_http'
+end
+
+template '/etc/apache2/sites-available/django.conf' do
+  source 'django.conf.erb'
+  action :create
+  owner 'root'
+  group 'root'
+  mode '0644'
+end
+
+execute 'disable 000-defautl.conf' do
+  command 'a2dissite 000-default.conf'
+end
+
+service 'apache2' do
```

Comment:
you don't need to reload twice. you can disable the default, enable the django and then reload after all